CHANGELOG.md

```diff
@@ -2,11 +2,71 @@
 
 ## Unreleased
 
-### Bugfixes
+### API
+
+* WARNING: GlobalOpenTelemetry trigger of autoconfiguration is now opt-in.
+  Previously, `GlobalOpenTelemetry.get` triggered autoconfiguration
+  if `opentelemetry-sdk-extension-autoconfigure` was detected on the classpath. That behavior is now
+  opt-in by setting environment variable `OTEL_JAVA_GLOBAL_AUTOCONFIGURE_ENABLED=true`.
```

Comment:
IIRC, `GlobalOpenTelemetry.get()` now throws an exception if the instance hasn't been set, correct? Worth calling out here as well?

Reply:
ah, yeah...had forgotten the exact case. at any rate...worth calling any of that out?

Reply:
I'd say no because none of that has changed.